### PR TITLE
CLOUD-444 added delete user operation

### PR DESCRIPTION
**Pull request by oleewere**
@bihartomi or @doktoric 
depends on: https://github.com/sequenceiq/cloudbreak/pull/739

### CODE REVIEW

File: app/static/messages.properties [hunk 1]
```diff
@@ -28,6 +28,8 @@
     "template_ssh_location_invalid": "ssh location field is invalid. example - 0.0.0.0/0",
     "user_already_exist": "user already exists",
     "user_form_admin_success" : "User permission has been updated successfully. (user: '{0}')",
+    "user_form_delete_success" : "User '{0}' deleted successfully.",
```
Comment (lpuskas) [inline app/static/messages.properties]: The message should be: "User '{0}' has been successfully deleted.

File: app/static/messages.properties [hunk 2]
```diff
@@ -28,6 +28,8 @@
...
+    "user_form_invite_success" : "User '{0}' invited successfully.",
```
Comment (lpuskas) [inline app/static/messages.properties]: User '{0}' successfully invited

File: app/static/messages.properties (hunk 1 above)
Comment (doktoric) [inline app/static/messages.properties]: User permission has been successfully updated

Comment (doktoric) [inline app/static/messages.properties]: user  with '{0}' name already exists

**Pull request closed without merging**